src/pages/PrivateNotes/PrivateNotesListPage.js

```diff
@@ -0,0 +1,158 @@
+     * @param {Object} item
+     * @param {Number} index
+     * @returns {JSX}
+     */
+    function getMenuItem(item, index) {
+        const keyTitle = item.translationKey ? translate(item.translationKey) : item.title;
+
+        return (
+            <OfflineWithFeedback
+                key={`${keyTitle}_${index}`}
+                pendingAction={item.pendingAction}
+                errorRowStyles={styles.ph5}
+                onClose={item.dismissError}
+                errors={item.errors}
+            >
+                <MenuItem
+                    title={keyTitle}
+                    icon={item.icon}
+                    iconType={CONST.ICON_TYPE_WORKSPACE}
+                    onPress={item.action}
+                    shouldShowRightIcon
+                    fallbackIcon={item.fallbackIcon}
+                    brickRoadIndicator={item.brickRoadIndicator}
+                />
+            </OfflineWithFeedback>
+        );
+    }
+
+    /**
+     * Returns a list of private notes on the given chat report
+     * @returns {Array} the menu item list
+     */
+    const privateNotes = useMemo(() => {
+        const privateNoteBrickRoadIndicator = (accountID) => (!_.isEmpty(lodashGet(report, ['privateNotes', accountID, 'errors'], '')) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '');
+        return _.chain(lodashGet(report, 'privateNotes', {}))
+            .map((privateNote, accountID) => ({
+                title: Number(lodashGet(session, 'accountID', null)) === Number(accountID) ? 'My note' : lodashGet(personalDetailsList, [accountID, 'login'], ''),
+                icon: UserUtils.getAvatar(lodashGet(personalDetailsList, [accountID, 'avatar'], UserUtils.getDefaultAvatar(accountID)), accountID),
+                iconType: CONST.ICON_TYPE_AVATAR,
+                action: () => Navigation.navigate(ROUTES.getPrivateNotesViewRoute(report.reportID, accountID)),
+                brickRoadIndicator: privateNoteBrickRoadIndicator(accountID),
+            }))
+            .value();
+    }, [report, personalDetailsList, session]);
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <FullPageNotFoundView
+                shouldShow={_.isEmpty(report.reportID) || (!report.isLoadingPrivateNotes && network.isOffline && _.isEmpty(lodashGet(report, 'privateNotes', {})))}
+                onBackButtonPress={() => Navigation.goBack()}
+            >
+                <HeaderWithBackButton
+                    title={translate('privateNotes.title')}
+                    shouldShowBackButton
+                    onCloseButtonPress={() => Navigation.dismissModal()}
+                    onBackButtonPress={() => Navigation.goBack()}
+                />
+                {report.isLoadingPrivateNotes && _.isEmpty(lodashGet(report, 'privateNotes', {})) ? (
+                    <FullScreenLoadingIndicator />
+                ) : (
+                    _.map(privateNotes, (item, index) => getMenuItem(item, index))
```

Comment:
Coming from https://github.com/Expensify/App/issues/27590
The notes list didn't take up the full screen height, which resulted in offline indicator not being positioned at the bottom
We resolved this by adding a ScrollView (in case there are more notes that fit on the screen) with `flexGrow: 1` style applied